package.json

```diff
@@ -15,13 +15,17 @@
   },
   "devDependencies": {
     "chai": "^3.5.0",
+    "eslint": "^4.11.0",
+    "eslint-config-airbnb-base": "^12.1.0",
+    "eslint-plugin-import": "^2.8.0",
     "mocha": "^2.5.3",
     "sinon": "^2.3.6"
   },
   "scripts": {
     "start": "node --max_old_space_size=350 index.js",
     "install": "./installModules.sh",
-    "test": "mocha $(git ls-files | grep -E \"test.js$\")"
+    "test": "mocha $(git ls-files | grep -E \"test.js$\")",
+    "lint": "eslint bot.js"
```

Comment:
Just saw this.
`eslint --ignore-pattern node_modules *.js modules/**/*.js`

Reply:
I'd like to move to that, but I'd rather have it run on the files it passes on for now and add the remaining files later.

Incrementally adding which files it lints allows us to get some signal from travis for some changes without having to fixup everything yet (which I'm not up to at the moment).